src/Sylius/Bundle/WebBundle/Behat/WebContext.php

```diff
@@ -888,4 +888,23 @@ public function iViewDeletedElements()
     {
         $this->clickLink('Show deleted');
     }
-}
+    
+    /**
+     * @Then I should see table sorted by lastName column
+     */
+    public function iShouldSeeTableSortedByLastName()
+    {
+        $allNames = $this->getSession()->getPage()->findAll('css', '#orders > tbody > tr > td > p');
```

Comment:
I think you can make this method more generic by passing a argument to this method which the id of the table.